wpilibc/athena/src/USBCamera.cpp

```diff
@@ -54,8 +54,9 @@ unsigned int USBCamera::GetJpegSize(void* buffer, unsigned int buffSize) {
     } else if (t == 0xda) {
       // SOS marker. The next two bytes are a 16-bit big-endian int that is
       // the length of the SOS header, skip that
-      unsigned int len = (((unsigned int)(data[pos + 2] & 0xff)) << 8 |
-                          ((unsigned int)data[pos + 3] & 0xff));
+      unsigned int len = (static_cast<unsigned int>(data[pos + 2]) & 0xff)
+                             << 8 |
+                         (static_cast<unsigned int>(data[pos + 3]) & 0xff);
```

Comment:
I was wondering why the casting is done two different ways for each byte. The change below does the same thing.